scene/2d/physics_body_2d.cpp

```diff
@@ -1271,14 +1271,10 @@ Vector2 KinematicBody2D::move_and_slide(const Vector2 &p_linear_velocity, const
 					}
 				}
 
-				if (p_stop_on_slope && is_on_slope) {
-					motion = motion.slide(p_floor_direction);
-					lv = lv.slide(p_floor_direction);
-				} else {
 					Vector2 n = collision.normal;
 					motion = motion.slide(n);
 					lv = lv.slide(n);
-				}
+
```

Comment:
`clang-format` is complaining about too many tabs here (as well as the blank line). https://travis-ci.org/godotengine/godot/jobs/423337098#L533

Reply:
fixed